### PR TITLE
7441: definition of done

### PR DESCRIPTION
**Pull request by kkoskelin**
package updates, code smells fixed

### CODE REVIEW

File: shared/src/business/entities/trialSessions/TrialSession.js [hunk 1]
```diff
@@ -130,84 +130,46 @@ TrialSession.PROPERTIES_REQUIRED_FOR_CALENDARING = {
...
-        is: TRIAL_SESSION_PROCEEDING_TYPES.remote,
-        otherwise: joi.allow('').optional(),
-        then: joi.when('sessionType', {
-          is: [SESSION_TYPES.special, SESSION_TYPES.motionHearing],
-          otherwise: joi.required(),
-          then: joi.allow('').optional(),
-        }),
-      }),
-    }),
+    passwoasdfasdfrd: stringRequiredForRemoteProceedings,
```
Comment (rachaelparris) [inline shared/src/business/entities/trialSessions/TrialSession.js]: ?

Comment (kkoskelin) [reply shared/src/business/entities/trialSessions/TrialSession.js]: whooops now, fixing

File: shared/src/business/entities/trialSessions/TrialSession.js [hunk 2]
```diff
@@ -130,84 +130,46 @@ TrialSession.PROPERTIES_REQUIRED_FOR_CALENDARING = {
...
-        is: TRIAL_SESSION_PROCEEDING_TYPES.remote,
-        otherwise: joi.allow('').optional(),
-        then: joi.when('sessionType', {
-          is: [SESSION_TYPES.special, SESSION_TYPES.motionHearing],
-          otherwise: joi.required(),
-          then: joi.allow('').optional(),
-        }),
-      }),
-    }),
+    meetingIdPhone: stringRequiredForRemoteProceedings,
```
Comment (rachaelparris) [inline shared/src/business/entities/trialSessions/TrialSession.js]: Why the name change?

Comment (kkoskelin) [reply shared/src/business/entities/trialSessions/TrialSession.js]: fast fingers and negligence. addressing.

Comment (kkoskelin) [reply shared/src/business/entities/trialSessions/TrialSession.js]: tbh I was lengthening the names so the blocks would wrap in the same manner to ensure they were indeed all the same block. Then I forgot to fix them.

**Review by rachaelparris — APPROVED** [review]

**Comment by matthopson** [comment]
I'm a little confused. This is 7441?

**Comment by kkoskelin** [comment]
> I'm a little confused. This is 7441?

@matthopson Apologies for not explaining further: as part of definition of done for 7441, simply reviewing package updates and code smells from SonarCloud.  The changes here are code smells reported recently, but not necessarily part of 7441 work. Just keeping the floor swept clean. :)

**Comment by matthopson** [comment]
Gotcha, thanks. Looked related to 7443, and there was nothing else identifiable as 7441 work, so I was scratching my head :P

**Pull request merged**